regression-test/suites/query_p0/sql_functions/string_functions/test_compress_varchar.groovy

```diff
@@ -0,0 +1,91 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite("test_compress_varchar") {
+    sql "drop table if exists compress_varchar;"
+    sql """
+        create table compress_varchar(rowid int, vc1 varchar(1), vc2 varchar(2),
+        vc3 varchar(3),vc4 varchar(4),vc5 varchar(5),vc6 varchar(6),vc7 varchar(7),
+        vc8 varchar(8),vc9 varchar(9),vc10 varchar(10),vc11 varchar(11),vc12 varchar(12),
+        vc13 varchar(13),vc14 varchar(14),vc15 varchar(15))
+        distributed by hash(rowid) buckets 5 properties('replication_num'='1');
+    """
+    sql """
+        insert into compress_varchar values(1,'a','ab','abc','abcd','abcde','abcdef','abcdefg','abcdefgh','abcdefghi','abcdefghij','abcdefghijk','abcdefghijkl','abcdefghijklm','abcdefghijklmn','abcdefghijklmno');
```

Comment:
If possible, some abnormal cases(such as empty value, null, or even some out of range values...) could be added.

Reply:
abnormal case is tested in ut. UT will generated random bytes with random length.